fintech-examples/fintech-server/pom.xml

```diff
@@ -59,6 +59,31 @@
             <version>5.1.7.RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.25.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.6</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.0.1</version>
```

Comment:
Can we move all these versions to dependency management in main `pom.xml`?

Reply:
I was not sure and did not want to bring fintech dependencies to main pom. But ok, I do.